CoreFoundation/Base.subproj/CFUtilities.c

```diff
@@ -1494,14 +1494,19 @@ CF_EXPORT Boolean _CFExtensionIsValidToAppend(CFStringRef extension) {
 
 #if DEPLOYMENT_RUNTIME_SWIFT
 
+#if DEPLOYMENT_TARGET_FREEBSD
```

Comment:
I know you listed as a known issue, but we'll have to resolve this before merging. It's used in quite a few places.